### PR TITLE
add travis ci

### PR DESCRIPTION
**Pull request by edisonxiang**
add travis ci fo this repository.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+ci-bot
```
Comment (m1093782566) [inline .gitignore:1]: Any reason why we should ignore ci-bot? Do you mean the ci-bot binary file?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
-# github robot
-github robot
+# ci-bot
+[![Go Report Card](https://goreportcard.com/badge/github.com/Huawei-PaaS/ci-bot)](https://goreportcard.com/badge/github.com/Huawei-PaaS/ci-bot)
+[![Build Status](https://travis-ci.org/Huawei-PaaS/ci-bot.svg?branch=master)](https://travis-ci.org/Huawei-PaaS/ci-bot)
+[![LICENSE](https://img.shields.io/badge/license-Apache%202-blue.svg)](https://github.com/Huawei-PaaS/ci-bot/blob/master/LICENSE)
+
+This repository houses ci bot which aims to provide more simple and rich features for the github project users.
```
Comment (m1093782566) [inline README.md]: What about outline the usage of ci bot?

This repository houses CI robot which will run your tests, add your labels, and merge your codes.

**Comment by m1093782566** [comment]
cc @sids-b

File: .gitignore (hunk 1 above)
Comment (edisonxiang) [reply .gitignore:1]: Yep, the ci-bot binary file. we should not keep it in this repo.

File: README.md (hunk 2 above)
Comment (edisonxiang) [reply README.md]: Thanks, I will update it.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.PHONY: all build ci-bot clean
```
Comment (m1093782566) [inline Makefile:1]: Do we need to create a docker image for ci-bot and then push it to a docker registry, e.g. dockerhub or Huawei cloud swr?

Comment (edisonxiang) [reply Makefile:1]: It is ok for me:)  which docker registry is suitable for us?
what about docker.io/Huawei-PaaS/ci-bot ?

Comment (m1093782566) [reply Makefile:1]: docker.io/Huawei-PaaS/ci-bot sounds good to me. Besides, we should consider Huawei Cloud SWR as well. Eventually, we will use Huawei Cloud SWR :)

I think we can add docker images support in follow-ups.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,28 @@
...
+
+## Build
+
+Before you are getting started, you must have [Go](https://golang.org/) already installed in your local machine.
```
Comment (sids-b) [inline README.md]: Before you are getting started, you must have -> Before you get started, make sure to have

**Comment by m1093782566** [comment]
LGTM overall. Please update the README, then we can merge this PR.

For docker images support, it should not be a blocker, we can do it in another PR.

File: .gitignore (hunk 1 above)
Comment (sids-b) [reply .gitignore:1]: Agree with @edisonxiang

File: Makefile (hunk 3 above)
Comment (edisonxiang) [reply Makefile:1]: Understood.

File: README.md (hunk 4 above)
Comment (edisonxiang) [reply README.md]: I will. thanks

**Comment by edisonxiang** [comment]
Done, Thanks @m1093782566 @sids-b

**Review by sids-b — APPROVED** [review]
LGTM. Thanks for your quick support :)

**Review by m1093782566 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**